include/bm/storage/bm_storage.h

```diff
@@ -119,8 +120,7 @@ struct bm_storage {
 	/**
 	 * @brief The event handler function.
 	 *
-	 * @note If set to NULL, no events will be sent.
-	 *       This field must be set manually.
+	 * @note This field must not be set manually.
```

Comment:
This should apply for the struct in total now, so these comments can be removed.

Reply:
That's fair, I agree.

Reply:
Updated.